app/pages/intl/test.tsx

```diff
@@ -0,0 +1,17 @@
+import type { NextPage } from 'next'
+import Title from './Title'
+import LanguageSwitcher from './LangugeSwitcher'
+import { ProvideLanguage } from '../../hooks/useLanguage'
+
+const Test: NextPage = () => {
+  return (
+    <ProvideLanguage>
+      <div className={'flex justify-center items-center h-screen'}>
```

Comment:
You can use just straight old `className="flex justify-center items-center h-screen"`; no need for `{' ... '}`

Reply:
Should we enforce this with the following eslint rules?
`'react/jsx-curly-brace-presence': ['error', 'never']`
`'jsx-quotes': ['error', 'prefer-single']`

Reply:
Yes this sounds like a good idea

There are probably still some more rules which are useful but missing, in such a case I'd suggest just quickly asking around if anyone's opposed to the rule and if not adding it real quick.